### PR TITLE
UPDATE BE

### PR DESCRIPTION
**Pull request by tiennguyen-developer**
1. Create Shoppify_service
2. Create Shoppify_data for MySql
3. Build Flyway connect MySQL

### CODE REVIEW

File: src/main/java/com/shoppify/entity/Role.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   @Column(name = "ID")
+   private long id;
+
+   @Column(name = "NAME")
+   private String name;
+
+   @Column(name = "DESCRIPTION")
+   private String description;
+
+   @ManyToOne
```
Comment (sn0509) [inline src/main/java/com/shoppify/entity/Role.java:27]: Change to Many-to-Many

File: src/main/java/com/shoppify/entity/User.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+   @Column(name="EMAIL", unique = true, nullable = false)
+   private String email;
+
+   @Column(name = "PASSWORD")
+   private String password;
+
+   @Column(name="IS_DELETED")
+   private boolean isDeleted;
+
+   @Column(name = "FULL_NAME")
```
Comment (sonnguyen-2000) [inline src/main/java/com/shoppify/entity/User.java]: These attributes should be separated into a new table with the name user_profile

File: src/main/java/com/shoppify/entity/Role.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+public class Role {
+   @Id
+   @GeneratedValue(strategy = GenerationType.IDENTITY)
+   @Column(name = "ID")
+   private long id;
+
+   @Column(name = "NAME")
+   private String name;
+
+   @Column(name = "DESCRIPTION")
```
Comment (sonnguyen-2000) [inline src/main/java/com/shoppify/entity/Role.java]: remove this addtribute

File: src/main/java/com/shoppify/entity/Address.java
Comment (sonnguyen-2000) [inline src/main/java/com/shoppify/entity/Address.java:1]: add new attribute "fullname" here

File: src/main/java/com/shoppify/entity/Role.java (hunk 1 above)
Comment (sonnguyen-2000) [inline src/main/java/com/shoppify/entity/Role.java]: remove @OnetoOne and change list<User> to Set<User> to ensure avoidance of duplication

Comment (tiennguyen-developer) [reply src/main/java/com/shoppify/entity/Role.java]: Một User sẽ có nhiều địa chỉ em

**Pull request closed without merging**